samples/subsys/usb/cdc_acm/overlay-composite-cdc-dfu.conf

```diff
@@ -1,5 +1,7 @@
 # Overlay file for composite configuration
 # CDC ACM + USB DFU
+#
+# This composite configuration may not work for Windows OS.
```

Comment:
Can you add the info in the commit message in this file as well?

Reply:
Done.